ROCK/GPIOBase.py

```diff
@@ -224,8 +248,8 @@ def validate_channel(self, channel):
                 self.validate_channel(c)
             return channel
         elif isinstance(channel, int):
-            if self.mode != BOARD:
-                raise ValueError("invalid channel given, mode is not BOARD, but channel is integer")
+            if self.mode != BOARD and self.mode != BCM:
```

Comment:
This is always false. Shouldn't it be `mode != BOARD or mode != BCM` ?

Reply:
You're correct, I think that's a mistake , I can push a fix up